cpp/oneapi/dal/table/backend/interop/host_homogen_table_adapter.cpp

```diff
@@ -350,10 +260,35 @@ host_homogen_table_adapter<Data>::host_homogen_table_adapter(const homogen_table
         return;
     }
 
+    size_t nFeatures = table.get_column_count();
+    size_t nRows = table.get_row_count();
```

Comment:
Problems:
1. Code style
2. Implicit singed to unsigned conversion
3. Missed const

Reply:
fixed